client/modules/IDE/components/FileNode.jsx

```diff
@@ -156,6 +168,8 @@ export class FileNode extends React.Component {
                   ref={(element) => { this[`fileOptions-${this.props.id}`] = element; }}
                   tabIndex="0"
                   onClick={this.toggleFileOptions}
+                  onBlur={this.blurComponent}
```

Comment:
could you call this function `onBlurComponent` rather than `blurComponent`?

Reply:
Yes! Thank you for the feedback.